src/main/java/seedu/bigpp/parser/Parser.java

```diff
@@ -15,6 +16,15 @@ public class Parser {
 
     public Command parseCommand(String userInput) {
 
+        String[] inputList = userInput.split(" ", 2);
```

Comment:
Good choice to have this step in the parseCommand() method instead of the individual parseViewerCommand() and parseBuilderCommand(). Shortens code and makes it neater